tests/e2e/tests/regression/browser/ttl-format.e2e.ts

```diff
@@ -1,54 +1,68 @@
-    .beforeEach(async () => {
+    .beforeEach(async() => {
         await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
     })
-    .afterEach(async () => {
+    .afterEach(async() => {
         //Clear and delete database
         for (let i = 0; i < keysData.length; i++) {
             await browserPage.deleteKey();
         }
         await deleteDatabase(ossStandaloneConfig.databaseName);
     })
+test('Verify that user can see TTL in the list of keys rounded down to the nearest unit', async t => {
+    //Create new keys with TTL
+    await t.click(cliPage.cliExpandButton);
+    for (let i = 0; i < keysData.length; i++) {
+        await t.typeText(cliPage.cliCommandInput, COMMANDS_TO_CREATE_KEY[keysData[i].textType](keysData[i].keyName), {paste: true});
+        await t.pressKey('enter');
+        await t.typeText(cliPage.cliCommandInput, `EXPIRE ${keysData[i].keyName} ${ttlForSet[i]}`, {paste: true});
+        await t.pressKey('enter');
+    }
+    await t.click(cliPage.cliCollapseButton);
+    //Refresh Keys in Browser
+    await t.click(browserPage.refreshKeysButton);
+    //Check that Keys has correct TTL value in keys table
+    for (let i = 0; i < keysData.length; i++) {
+        const ttlValueElement = Selector(`[data-testid="ttl-${keysData[i].keyName}"]`);
+        await t.expect(ttlValueElement.textContent).contains(ttlValues[i], `TTL value in keys table is ${ttlValues[i]}`);
+    }
+});
 test
-    .meta({ rte: rte.standalone })
-    ('Verify that user can see TTL in the list of keys rounded down to the nearest unit', async t => {
-        //Create new keys with TTL
-        await t.click(cliPage.cliExpandButton);
-        for (let i = 0; i < keysData.length; i++) {
-            await t.typeText(cliPage.cliCommandInput, COMMANDS_TO_CREATE_KEY[keysData[i].textType](keysData[i].keyName), { paste: true });
-            await t.pressKey('enter');
-            await t.typeText(cliPage.cliCommandInput, `EXPIRE ${keysData[i].keyName} ${ttlForSet[i]}`, { paste: true });
-            await t.pressKey('enter');
-        }
-        await t.click(cliPage.cliCollapseButton);
-        //Refresh Keys in Browser
+    .after(async() => {
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    ('Verify that Key is deleted if TTL finishes', async t => {
+        // Create new key with TTL
+        await browserPage.addStringKey(keyName, 'test', '3');
         await t.click(browserPage.refreshKeysButton);
-        //Check that Keys has correct TTL value in keys table
-        for (let i = 0; i < keysData.length; i++) {
-            const ttlValueElement = Selector(`[data-testid="ttl-${keysData[i].keyName}"]`);
-            await t.expect(ttlValueElement.textContent).contains(ttlValues[i], `TTL value in keys table is ${ttlValues[i]}`);
-        }
+        await t.expect(await browserPage.isKeyIsDisplayedInTheList(keyName)).ok('Added key');
+        await t.wait(3000);
```

Comment:
Reason?

Reply:
Added a comment why wait is needed - I am checking that after TTL finishing , key is not displayed in the application

Reply:
So please do instead `await t.expect(browserPage.refreshKeysButton).visible('visibility of ttl' , { timeout: 5000 }` - something on that spirit, it's better approach. Waits within tests are highly not recommended.

Reply:
We don't auto-refresh list of keys. For this reason we need to wait for 3 seconds, than click on refresh button and only after it check key lack in the app. I have tried the way you spicified, but in doesn't work such way. If you have any concerns please let me know

Reply:
What about implement a function of for loop with x tries?

Reply:
As we discussed, let's add explanation here

Reply:
As we discussed, I have added a loop with refreshing and validating reduced TTL value in the test